includes/class-wi-volunteer-management.php

```diff
@@ -81,6 +81,7 @@ public function __construct() {
 
 		// Load the public hooks
 		$this->define_public_hooks();
+      $this->define_widget_hooks();
```

Comment:
**Pull request updated**:
I moved the action to initialize the widget into the define_public_hooks function, and moved the register_widget function from widget/class-widget.php to frontend/class-public.php

Thanks for the feedback!